Updated Wordpress tutorial

activatedgeek opened the pull request:
The new tutorial now uses official Wordpress Docker Image.

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "patch-1" git@github.com:activatedgeek/compose.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

GordonTheTurtle commented:
Please sign your commits following these rules:
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#sign-your-work
The easiest way to do this is to amend the last commit:
~~~console
$ git clone -b "patch-1" git@github.com:activatedgeek/compose.git somewhere
$ cd somewhere
$ git commit --amend -s --no-edit
$ git push -f
~~~

Ammending updates the existing PR. You **DO NOT** need to open a new one.

activatedgeek commented:
@dnephin When can I expect this to go live?

dnephin commented:
@activatedgeek I don't have any estimate for you unfortunately.

cc @londoncalling for docs review

londoncalling commented on docs/wordpress.md:
```diff
@@ -22,116 +22,47 @@ with Docker containers. This quick-start guide demonstrates how to use Compose t
 ### Build the project
 
-With those four new files in place, run `docker-compose up` from your project directory. This will pull and build the needed images, and then start the web and database containers.
+With those four new files in place, run `docker-compose up -d` from your project directory. This will pull and build the needed images, and then start the wordpress and database containers.
```
@dnephin something seems to be missing here, because as written:
* this doesn't make sense anymore "with those four new files in place" (which 4 files? because now we don't tell them to create them)
*  it doesn't run

I see that the `docker-compose.yml` file is calling different DB and Wordpress images, defining different volumes, wordpress, etc. but how is the .yml supposed to find what it needs?  Doesn't seem to be working for me.

Also there doesn't seem to be a reference point for `MariaDB` comment. Where would I see that referenced? In one of the containers? In another config file?

londoncalling commented:
@dnephin @activatedgeek please see my comments inline on the markdown file; I can't get the sample app to run by following the tutorial as is now

activatedgeek commented:
Hey @londoncalling, I updated the misleading old text in the document. Also I just ran a fresh project, created a folder, adding the docker compose file mentioned in the document and it works. I have done absolutely no modifications whatsoever to the `wordpress` or `mysql` docker images.

I have added an extra note as the containers take a couple of minutes to bootstrap on the first load. Once the setup is complete it should work as claimed.

londoncalling commented:
@activatedgeek yes it works now. Thanks for updating the wording, it reads very nicely. As for it not running before, I believe that was caused by some problems I was having with some in-development versions of Docker. All good now. LGTM! @dnephin
